sites/all/modules/custom/ciandt_wechat_vacation/ciandt_wechat_vacation.module

```diff
@@ -442,7 +442,7 @@ function ciandt_wechat_vacation_form_vocation_node_form_alter(&$form, &$form_sta
   if (isset($user->name) && isset($options[$user->name])) {
     unset($options[$user->name]);
   }
-  $output = '<div class="popup"><div class="popup-top"><span class="close-btn">X</span></div><ul>';
+  $output = '<div class="popup"><div class="popup-top"><span class="close-btn">X</span></div><div class="approver-autocomplete"><input id="approver-autocomplete-search" class="form-text"/></div><ul class="initial-approver-list">';
```

Comment:
Let's break the long code string to new line, Ex. $output .= '<div class="approver-autocomplete">....';

Reply:
@jieyyal  Fixed, please check.